cmd/cli/plugin/package/package_available_get.go

```diff
@@ -113,20 +115,20 @@ func packageAvailableGet(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func getValuesSchema(cmd *cobra.Command, args []string, kc kappclient.Client) error {
-	if pkgVersion == "" {
+func getValuesSchemaForPackage(namespace, name, version string, kc kappclient.Client, writer io.Writer) error {
```

Comment:
We do not have a fake `cobra.Command` available, so replacing `cmd *cobra.Command` with `writer io.Writer` is to make it possible for unit tests to pass in other `io.Writer` (i.e., `bytes.Buffer`).